tests/baselines/reference/tsbuild/amdModulesWithOut/prepend-reports-deprecation-error.js

```diff
@@ -387,11 +387,46 @@ sourceFile:global.ts
       "./global.ts"
     ],
     "fileInfos": {
-      "../../lib/lib.d.ts": "3858781397-/// <reference no-default-lib=\"true\"/>\ninterface Boolean {}\ninterface Function {}\ninterface CallableFunction {}\ninterface NewableFunction {}\ninterface IArguments {}\ninterface Number { toExponential: any; }\ninterface Object {}\ninterface RegExp {}\ninterface String { charAt: any; }\ninterface Array<T> { length: number; [n: number]: T; }\ninterface ReadonlyArray<T> {}\ndeclare const console: { log(msg: any): void; };",
-      "./file0.ts": "3587416848-const myGlob = 20;",
-      "./file1.ts": "-10726455937-export const x = 10;",
-      "./file2.ts": "-13729954175-export const y = 20;",
-      "./global.ts": "1028229885-const globalConst = 10;"
+      "../../lib/lib.d.ts": {
+        "original": {
+          "version": "3858781397-/// <reference no-default-lib=\"true\"/>\ninterface Boolean {}\ninterface Function {}\ninterface CallableFunction {}\ninterface NewableFunction {}\ninterface IArguments {}\ninterface Number { toExponential: any; }\ninterface Object {}\ninterface RegExp {}\ninterface String { charAt: any; }\ninterface Array<T> { length: number; [n: number]: T; }\ninterface ReadonlyArray<T> {}\ndeclare const console: { log(msg: any): void; };",
+          "impliedFormat": 1
+        },
+        "version": "3858781397-/// <reference no-default-lib=\"true\"/>\ninterface Boolean {}\ninterface Function {}\ninterface CallableFunction {}\ninterface NewableFunction {}\ninterface IArguments {}\ninterface Number { toExponential: any; }\ninterface Object {}\ninterface RegExp {}\ninterface String { charAt: any; }\ninterface Array<T> { length: number; [n: number]: T; }\ninterface ReadonlyArray<T> {}\ndeclare const console: { log(msg: any): void; };",
+        "impliedFormat": "commonjs"
```

Comment:
I think impliedFormat encoding here seems wrong. its not module? we need better logic in builder for checking things as well as not bloating buildinfo with this. It quickly increases when you have lots of files in project like "owa"

Reply:
What would cause it to be `module`? What do you suggest for reducing the buildinfo size? We can keep `impliedNodeFormat` as `undefined` for AMD/UMD, but those are rarely used and we may decide to deprecate them in 6.0 anyway.

Reply:
I just looked into this a bit more and realized that this is anyways encoded as object in tsbuildinfo as it has `affectsGlobalScope` so this should be ok as is.. i will later look into if i can modify how the fileInfo is encoded and see if we can do anything differently but nothing for you to block on.